YR-[CIAC-6129]-fix-8-commands-to-work-with-a-single-FW (#25924)
demisto/content: Packs/PAN-OS/ReleaseNotes/1_16_21.md

#### Integrations

##### Palo Alto Networks PAN-OS

> - Added support for the **pan-os-apply-dns-signature-policy** command, to support a single firewall.

Reviewer: ```suggestion
- Fixed an issue where the ***pan-os-apply-dns-signature-policy*** command, didn't support a single firewall.
```
